Calculator.php

```diff
@@ -147,3 +132,26 @@ function info($result)
 {
     echo $result . PHP_EOL . PHP_EOL;
 }
+
+function noMathCommand($command)
+{
+    switch($command) {
+        case(QUIT):
+            $command = choose('Are you sure to wanna quit? Yes/No ', [AGREE, DEGREE]);
+
+            if ($command == AGREE) {
+            exit();
+            }
+
+            break;
+
+        case(INFO):
+            info((implode(' ;  ', AVAILABLE_COMMANDS)) . ' ;');
+
+            break;
+
+        case(HISTORY):
+            $history = file_get_contents('history.txt');
+            info($history);
```

Comment:
I believe it would be better to implement it as a separate commands

```suggestion
        case(QUIT):
            finish_app();

            break;
        case(INFO):
            show_info_block();

            break;
        case(HISTORY):
            show_history();

            break;
```